odk1-src/central-install-digital-ocean.rst

```diff
@@ -137,6 +137,19 @@ Once you do see it working, you'll want to set up your first Administrator accou
 
 Once you have one account, you do not have to go through this process again for future accounts: you can log into the website with your new account, and directly create new users that way.
 
+.. _central-install-digital-ocean-monitoring:
+
+Setting Up Monitoring
+---------------------
+
+The last thing you will want to do is to set up server monitoring. Alerts and monitoring are important because they can inform you of problems with your server before they affect your data collection project.
+
+You can find instructions for setting up alerts in the `DigitalOcean Documentation  <https://www.digitalocean.com/docs/monitoring/how-to/set-up-alerts/>`_.
+
+We strongly recommend creating an alert for Disk Utilization. A threshold of 90% is usually reasonable. By far the most common operations issue we see is servers running out of disk space as large photo files pile up. If your server runs entirely out of disk space, it can crash and become unresponsive. It is best to upgrade your storage plan before this happens.
+
+If you are familiar with server operations, you may wish to set up some other alerts: CPU usage and Memory Utilization are the most interesting remaining metrics. However, these are not as important or easily understandable as the Disk Utilization alert, so you may skip this if you're not sure what to do here.
```

Comment:
s/large photo files/large media attachments/. Matches the language we use earlier in the document.

I'd recommend the default CPU alert on DO. If you peg the CPU at 90% for 5 minutes, that's good to know. It's probably due to XLSForm conversion (and in the future Enketo).

> We also recommend creating an alert for CPU Utilization. A threshold of 90% is usually reasonable. If your server is regularly using a lot of CPU, it be good to upgrade your plan to make it more performant.

If you make the above change, the the next paragraph should only mention CPU usage. Either way, I'd drop the "easily understandable". I think it's pretty understandable.

Reply:
i will protest mildly that i don't think CPU utilization behaviour is easily understandable, and that noisy alerts pegging because of eg rare form conversions (esp once we have enketo integrated) are not an indication that a higher tier is necessary, or indeed that going to a higher tier will even resolve the alert. but i'll add this if you feel strongly about it.

Reply:
Talked a little bit about this with @issa-tseng and I agree with leaving it out for now. The argument is that it's likely to lead to false positives that people would ask about and would be hard to troubleshoot.